core/templates/dev/head/pages/header_js_libs.html

```diff
@@ -12,3 +12,5 @@
 <script src="/third_party/static/jquery-ui-touch-punch-0.3.1/jquery.ui.touch-punch-improved.js"></script>
 <script src="/third_party/static/headroom-js-0.9.4/headroom.min.js"></script>
 <script src="/third_party/static/headroom-js-0.9.4/angular.headroom.min.js"></script>
+<script src="/third_party/static/math-js-4.2.2/math.js"></script>
```

Comment:
only add `math.min.js`.

Reply:
Better use bundle argument in manifest.json's mathjs section to bundle this with other third_party js libs.

Reply:
Ok, I'll add only `math.min.js`.